src/settings/utils/updateClosingTypes.js

```diff
@@ -0,0 +1,8 @@
+export default (closingTypes, updatedType) => {
```

Comment:
I think this can just be a method in the `src/settings/LoanHistory/LoanHistoryForm.js`. No need for it to be in the utils now.